sql/schema/dsl.go

```diff
@@ -231,6 +231,12 @@ func NewView(name, def string) *View {
 	return &View{Name: name, Def: def}
 }
 
+// NewMaterialized creates a new materialized View.
+func NewMaterialized(name, def string) *View {
```

Comment:
```suggestion
func NewMaterializedView(name, def string) *View {
```
Keeps it more closed to SQL imho.